index.html

```diff
@@ -17,130 +17,205 @@
             <div class="hamburger" > <nav><img  src="./images/Hamburger.png">  </nav> </div>          
        </header>
 
```

Comment:
- [ ] I kindly suggest that you please fix the horizontal scroll on your desktop by making your web portfolio more responsive.

https://user-images.githubusercontent.com/52934128/194643605-a97f37cb-1b87-47fe-be02-dec573c4366c.mp4

Reply:
[OPTIONAL BUT REQUIRED]
- As suggested by the previous reviewer, The horizontal scroll is still not fully resolved, please fix this issue. I'm marking it as OPTIONAL because you did try working on it.